src/types/severa/workHour/severaResponseWorkHours.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Interface for Severa Response Work Hour.
+ */
+interface SeveraResponseWorkHours {
+  guid: string;
+  user: {
+    guid: string;
+    name: string;
+  };
```

Comment:
No need for the semi-colon here after the closing curly brackets, please remove all